android/src/main/java/com/reactnativemembrane/Utils.kt

```diff
@@ -46,15 +50,56 @@ internal fun listToRNArray(array: List<Any>): WritableArray {
   return res
 }
 
-internal fun mapToRNMap(map: Map<String, Any>): WritableMap {
+internal fun rtcOutboundStatsToRNMap(stats: RTCOutboundStats): ReadableMap{
+  val innerMap = Arguments.createMap()
+  innerMap.putDouble("bandwidth", if (stats.qualityLimitationDurations?.bandwidth != null) stats.qualityLimitationDurations!!.bandwidth else  0.0)
+  innerMap.putDouble("cpu", if (stats.qualityLimitationDurations?.cpu != null) stats.qualityLimitationDurations!!.cpu else  0.0)
```

Comment:
can't you do something like this?
`innerMap.putDouble("cpu", stats.qualityLimitationDurations?.cpu ?: 0.0)`
https://kotlinlang.org/docs/null-safety.html#elvis-operator

Reply:
Done.